src/StatusQ/Components/StatusListItem.qml

```diff
@@ -34,6 +34,9 @@ Rectangle {
 
     radius: 8
 
+    property string itemId: ""
```

Comment:
So this property only really makes sense in the search results. It shouldn't be part of `StatusListItem` by default because there's no natural id for any list item.

Can we move that inline to the `StatusListItem` used in `StatusSearchPopup` instead?
I'd also be fine with extracting that one into its own `StatusSearchResultItem` component.